workflow-service/frontend/Dockerfile

```diff
@@ -12,11 +15,10 @@ ARG FRONTEND_ENV=production
 
 ENV VUE_APP_ENV=${FRONTEND_ENV}
```

Comment:
What's the difference between production and development?

Reply:
Does it cause differences in the behavior. I'm just wondering.

Reply:
no, not now. Maybe we can remove it. I'm doing a local test currently with setting base url to `/` for production and docker setups.